<launch>
  <node pkg="cotesys_ros_grasping" type="move_arm_to_position_server" name="move_arm_to_position_server" output="screen">
    <remap from="left_move_arm_action" to="move_left_arm"/>  
    <remap from="right_move_arm_action" to="move_right_arm"/>  
    <param name="end_effector_rot_z" type="double" value=".7071" />
    <param name="end_effector_rot_x" type="double" value=".7071" />    
    <param name="end_effector_rot_w" type="double" value="0.0" />    
  </node>  
</launch>
